egs/librispeech/asr/simple_v1/mmi_att_transformer_train.py

```diff
@@ -231,14 +234,14 @@ def train_one_epoch(dataloader: torch.utils.data.DataLoader,
                     tb_writer: SummaryWriter,
                     num_epochs: int,
                     global_batch_idx_train: int):
-    """One epoch training and validation.
+    '''One epoch training and validation.
```

Comment:
I think it's a bad pattern to do some many formatting changes in a PR like this; it's very distracting from what's the actual contribution.

Reply:
Ok, reverted.